mkstage4.sh

```diff
@@ -80,7 +93,7 @@ shift;OPTIONS="$@"
 EXCLUDES="\
 --exclude=.bash_history \
 --exclude=dev/* \
---exclude=lost+found \
+--exclude=lost+found/* \
```

Comment:
Good question. And sorry I forgot to mention that in the commit message. The lost+found directory is part of the ext4 filesystem and it has some special properties. See [here](http://unix.stackexchange.com/a/18157/22172). Actually, it may be wise to include the files in the backup, in which case the line could be removed completely.